cmd/collect.go

```diff
@@ -85,16 +93,10 @@ func collectCommand(c *cli.Context) error {
 	}
 
 	if !cr.Exists {
-		logging.S().Errorw("no such testplan run", "run_id", id, "runner", runner)
+		logging.S().Errorw("no such testplan run", "run_id", runid, "runner", runner)
```

Comment:
This will leave a 0kb file behind, no? Shouldn't we delete it here?

Reply:
Added a delete on error.